ProjectSettings/InputManager.asset

```diff
@@ -294,7 +294,7 @@ InputManager:
     axis: 0
     joyNum: 0
   - serializedVersion: 3
-    m_Name: ControllerLeftStickX
```

Comment:
@keveleigh did you get to test this with the motion controllers?

I still plan on revisiting this to make sure we're clear of any conflicts.